src/test/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/GenerateComponentLocationAnalysisOperationIT.java

```diff
@@ -1,17 +1,17 @@
 package com.synopsys.integration.detect.workflow.componentlocationanalysis;
 
+import java.io.IOException;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
 import com.synopsys.integration.detect.battery.docker.provider.BuildDockerImageProvider;
 import com.synopsys.integration.detect.battery.docker.util.DetectCommandBuilder;
 import com.synopsys.integration.detect.battery.docker.util.DetectDockerTestRunner;
 import com.synopsys.integration.detect.battery.docker.util.DockerAssertions;
 import com.synopsys.integration.detect.configuration.DetectProperties;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
 
-import java.io.IOException;
-
-@Disabled
```

Comment:
Not sure why this was disabled. Trying to enable and run comprehensive build now.

Reply:
The test fails on assertions that existed before my assertions were added.
I will re-disable the test, merge, and then continue to investigate.